# Chunks 1 and 2 are already in log4cxx SVN, but version 0.10.0 does not contain them
--- src/main/cpp/inputstreamreader.cpp.orig	2011-01-16 10:11:26.000000000 +0100
+++ src/main/cpp/inputstreamreader.cpp	2011-01-16 01:12:20.000000000 +0100
@@ -15,6 +15,7 @@
  * limitations under the License.
  */
 
+#include <cstring>
 #include <log4cxx/logstring.h>
 #include <log4cxx/helpers/inputstreamreader.h>
 #include <log4cxx/helpers/exception.h>
--- src/main/cpp/socketoutputstream.cpp.orig	2011-01-16 10:11:20.000000000 +0100
+++ src/main/cpp/socketoutputstream.cpp	2011-01-16 01:15:07.000000000 +0100
@@ -15,6 +15,7 @@
  * limitations under the License.
  */
 
+#include <cstring>
 #include <log4cxx/logstring.h>
 #include <log4cxx/helpers/socketoutputstream.h>
 #include <log4cxx/helpers/socket.h>
# I should send them this patch
--- liblog4cxx.pc.in.orig	2011-01-16 20:19:22.000000000 +0100
+++ liblog4cxx.pc.in	2011-01-16 20:20:00.000000000 +0100
@@ -22,4 +22,5 @@
 Version: @VERSION@
 Libs: -L${libdir} -llog4cxx
 Cflags: -I${includedir}
+Requires.private: apr-util-1
 
